src/app/shared/components/sidebar-item/sidebar-item.component.html

```
@@ -26,13 +27,13 @@
   </div>
   } @else {<!-- Single menu item -->
   <a
-    [routerLink]="menuItem.route"
+    [routerLink]="menuItem().route"
     routerLinkActive="bg-blue-600 text-white"
     [routerLinkActiveOptions]="{ exact: false }"
     class="flex items-center gap-3 p-3 rounded-lg hover:bg-gray-700 transition-colors"
```

HTMLHint (1.5.0)

[warning] The id and class attribute values must be in lowercase and split by a dash.

(id-class-value)